cypress/e2e/ScraperLiteEdition/APIInfo.cy.js

```diff
@@ -0,0 +1,97 @@
+import { SCRAPER_BASE_URL } from '../../resources/data';
+
+describe('Scraper(Lite Edition) APIs Info page', () => {
+    before(() => {
+        cy.visit(SCRAPER_BASE_URL);
+    });
+
+    it('1. Test to "Accept Cookies" if not already accepted', () => {
+        cy.AcceptCookies();
+    });
+
+    it('2. Test to check for the "API Info" tab is present', () => {
+        cy.componentVisiblityCheck('#details-tab > .d-md-flex > span');
+    });
+
+    it('3. Test to click on the "API Info" tab', () => {
+        cy.get('#details-tab > .d-md-flex > span').click();
+    });
+
+    it('4. Test to click "Adanced Scraper API" link', () => {
+        cy.get('article > :nth-child(4) > a').click();
+        cy.document().should((doc) => {
```

Comment:
Remove nth child all over the page

Reply:
I have removed nth child for some it blocks, i kept only for "Advanced Scraper API" link as i can see there are two same links present in the page. i need to differentiate between them and i need to click both of them.